crates/uv-build-backend/src/metadata.rs

```diff
@@ -73,12 +79,14 @@ pub fn check_direct_build(source_tree: &Path, name: impl Display) -> bool {
             Ok(pyproject_toml) => pyproject_toml,
             Err(err) => {
                 debug!(
-                    "Not using uv build backend direct build of {name}, no pyproject.toml: {err}"
+                    "Not using uv build backend direct build of `{name}`, \
```

Comment:
Maybe?

```suggestion
                    "Not using uv build backend direct build for source tree `{name}`, \
```

Separately, should we just use the unsimplified display here? A relative path doesn't seem super userful?

Reply:
We do use the other display in some other debug level logs. I don't have strong feelings.